scripts/get-version.ps1

```diff
@@ -2,7 +2,7 @@
 $ErrorActionPreference="Stop"
 
 git update-index -q --refresh
-git diff-files --quiet | Out-Null
+git diff-files -- . ':!**/go.mod' ':!**/go.sum' --quiet | Out-Null
```

Comment:
Surprised these matching expressions don't need to change for windows. Would've expected something more like `':!**\\go.mod'`

Reply:
🤷‍♂ but it works - I was able to publish a windows binary just fine